packages/components/music/UploadMusicModal/UploadTrack.tsx

```diff
@@ -50,19 +53,34 @@ export const UploadTrack: React.FC<Props> = ({ onUploadDone }) => {
   const selectedNetwork = useSelectedNetworkInfo();
   const selectedWallet = useSelectedWallet();
   const userId = selectedWallet?.userId;
-  const [isNotEnoughFundModal, setNotEnoughFundModal] = useState(false);
   const { uploadFilesToPinata, ipfsUploadProgress } = useIpfs();
-  const { makePost, canPayForPost, isProcessing, step, setStep } =
-    useFeedPosting(selectedNetwork?.id, userId, PostCategory.MusicAudio, () => {
-      // Timeout here to let a few time to see the progress bar "100% Done"
-      setTimeout(() => {
-        setIsUploadLoading(false);
-        setIsProgressBarShown(false);
-        onUploadDone();
-      }, 1000);
-    });
+  const postCategory = PostCategory.MusicAudio;
+  const {
+    makePost,
+    canPayForPost,
+    isProcessing,
+    freePostCount,
+    step,
+    setStep,
+  } = useFeedPosting(selectedNetwork?.id, userId, postCategory, () => {
+    // Timeout here to let a few time to see the progress bar "100% Done"
+    setTimeout(() => {
+      setIsUploadLoading(false);
+      setIsProgressBarShown(false);
+      onUploadDone();
+    }, 1000);
+  });
   const [isUploadLoading, setIsUploadLoading] = useState(false);
   const [isProgressBarShown, setIsProgressBarShown] = useState(false);
+  const { postFee } = useFeedPostFee(selectedNetwork?.id, postCategory);
```

Comment:
this is already returned by `useFeedPosting` in `publishingFee`

Reply:
Fixed